charts/sysdig/templates/daemonset.yaml

```diff
@@ -229,6 +233,9 @@ spec:
         {{- if (and .Values.ebpf.enabled .Values.slim.enabled) }}
         - name: bpf-probes
           emptyDir: {}
+	- name: sys-tracing
+          hostPath:
+	    path: /sys/kernel/debug
```

Comment:
It looks there is some weird indentation here? Maybe tabs vs spaces?

Reply:
Thanks for the review.  I will fix the indentation, update the version and CHANGELOG, and resubmit the PR.
Thanks!